### DISCUSSION TITLE
Is it possible to use the container without an internet connection?

### DISCUSSION THREAD

**Discussion by sagbdev** [open-webui/open-webui#820] [Q&A] [answered]
I have an unreliable internet connection. Today, I tried to use the Open WebUI container from localhost but was unable to connect to the container because it requires an internet connection to start. Is there a way for me to run it totally offline? I've been using Ollama as an alternative to ChatGPT when my internet connection is down.

**Comment by GanizaniSitara** [comment]
Can't speak for the container, but I'm just working through installing offline, or rather without hugging face access. Build ok, but even though I've copied in the model all-MiniLM-L6-v2 into place it still seems to want to connect to hugging face, will need bit of troubleshooting.

**Comment by jannikstdl** [comment]
Yes, you can run the Open WebUI locally. without interner connection. Of course you need to have a connection to pull the image.

**Reply by z1mpleee** [reply]
After connecting to the internet, I can use it normally. However, when I switch the server back to the intranet environment, the login page can be displayed successfully. But after successful login, it shows a blank page.
The console shows that all requests are responding normally, but there is one error.
![WXWorkLocal_20240326152430](https://github.com/open-webui/open-webui/assets/45474697/47ea0475-1521-4433-bc8e-fc9eac7f7072)

**Comment by sagbdev** [comment, marked as answer]
I just created a podman quadlet and it seems to have been fixed for a while now, I'm closing the discussion.
